src/webrtc/datachannel.ts

```diff
@@ -249,7 +249,14 @@ export class DataChannelClass implements DataChannel {
         setTimeout(this.conjestionControlSendHandler, 20);
       } else {
         if(!this.toPeerDataQueue_.isHandling()) {
-          this.toPeerDataQueue_.setHandler(this.handleSendDataToPeer_);
+          // This processes one block from the queue, which (in Chrome) is
```

Comment:
If I'm reading this correctly, you can remove the `if` check because the queue will never have a handler. Is that correct?